packages/js/src/types.ts

```diff
@@ -1,7 +1,111 @@
-export interface PaginatedResponse<T = unknown> {
+export enum NotificationStatus {
+  READ = 'read',
+  SEEN = 'seen',
+  UNREAD = 'unread',
+  UNSEEN = 'unseen',
+}
+
+export enum NotificationButton {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary',
+}
+
+export enum NotificationActionStatus {
+  PENDING = 'pending',
+  DONE = 'done',
+}
+
+export enum ActorType {
+  NONE = 'none',
+  USER = 'user',
+  SYSTEM_ICON = 'system_icon',
+  SYSTEM_CUSTOM = 'system_custom',
+}
+
+export enum CtaType {
+  REDIRECT = 'redirect',
+}
+
+export type Session = {
+  token: string;
+  profile: {
```

Comment:
```suggestion
  subscriber:
```

Reply:
I responded in another PR: https://github.com/novuhq/novu/pull/5665/files#r1630782252. I'll create a ticket for that.

Reply:
https://linear.app/novu/issue/COM-33/refactor-the-session-interface